app/grandchallenge/algorithms/models.py

```diff
@@ -84,6 +89,18 @@ class Algorithm(UUIDModel, TitleSlugDescriptionModel):
             "terms of usage, define them here."
         ),
     )
+    result_template = models.TextField(
+        blank=True,
```

Comment:
Should we add a default to help people get going? `default="<pre>{{ result_dict }}</pre>"`?

Reply:
Good idea